infrastructure/smart-contracts/scripts/tiny-keys/tinyKeysAirdropDeployer.mjs

```diff
@@ -10,18 +10,27 @@ async function main() {
 
     // get the deployer
     const [deployer] = (await ethers.getSigners());
-    const deployerAddress = await deployer.getAddress();
```

Comment:
We will want to print the deployer key just for sanity check.

Reply:
Done!